.infrastructure/confgiMap.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: todoapp-config
+  namespace: todoapp
+data:
+  PYTHONUNBUFFERED: "1"
+  
```

Comment:
Last line isn't empty and EoF doesn't work
![image](https://github.com/mate-academy/devops_todolist_kubernetes_task_7_configuration_management/assets/87720583/a0f29c57-2e6d-4e20-b53b-37e45191697c)